### PR TITLE
fix: decouple report styles

### PR DESCRIPTION
**Pull request by GusBaamonde**
#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

File: assets/scss/templates/_reportes.scss [hunk 1]
```diff
@@ -173,6 +173,35 @@
   }
 }
 
+/*********/
```
Comment (andresmoschini) [inline assets/scss/templates/_reportes.scss]: What is the meaning of this separator?

File: assets/scss/templates/_reportes.scss [hunk 2]
```diff
@@ -240,8 +269,8 @@
 
 /* Top ranking */
 .wrapper-reports-box {
-  max-width: $dp-wrapper-max-width;
-  margin: $dp-spaces--lv4 auto;
+  /*max-width: $dp-wrapper-max-width;*/
+  /*margin: $dp-spaces--lv4 auto;*/
```
Comment (andresmoschini) [inline assets/scss/templates/_reportes.scss]: What do you think about removing commented code?

Comment (GusBaamonde) [reply assets/scss/templates/_reportes.scss]: If I was just generating the version to be able to do tests on the webapp.
Then I delete everything and the word neat

File: assets/scss/templates/_reportes.scss (hunk 1 above)
Comment (GusBaamonde) [reply assets/scss/templates/_reportes.scss]: If I was just generating the version to be able to do tests on the webapp.
Then I delete everything and the word neat

File: assets/scss/templates/_reportes.scss [hunk 3]
```diff
@@ -173,6 +173,35 @@
...
 
+/*********/
+
+.box-border--bottom {
+  position: relative;
+
+  &::after {
+    content: '';
+    position: absolute;
+    border-bottom: 3px solid #33ad73;
```
Comment (cbernat) [inline assets/scss/templates/_reportes.scss]: You have a rogue color here `#33ad73`

**Pull request merged**